src/detections/filters/wrapper.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useProps } from 'detections/filters/use-props'
+
+export const Wrapper = ({ children }) => <Grid {...useProps()}>
+  {children}
+</Grid>
+
+Wrapper.propTypes = { children: PropTypes.element.isRequired }
```

Comment:
Same here.

Reply:
And all components receiving the `Children` prop.

Reply:
done